DashKit/DashPumpManager.swift

```diff
@@ -19,10 +19,6 @@ public protocol PodStatusObserver: class {
 
 public class DashPumpManager: PumpManager {
     
-    public func acknowledgeAlert(alertIdentifier: DeviceAlert.AlertIdentifier) {
```

Comment:
I think we want to remove the protocol default for this and keep the local stub, right?

Reply:
Oops, nevermind. Missed it below.